minetest/minetest
# [NO SQUASH] content_cao: Don't expire visuals when unnecessary + nametag cleanups

1) fixes #6572

2) cleanup & removal of cmd that was deprecated in 9eee3c3f465c071bb9908749cf48be3c131a1bdf

## To do

This PR is Ready for Review.

## How to test

1) Basic testing code, there should be no player animation reset:
```lua
local function x()
	local player = core.get_player_by_name("singleplayer")
	player:set_properties(player:get_properties())
	core.after(2, x)
end
core.after(2, x)
```
2) Use the entity spawner & prop editor in devtest to confirm nametag functionality.
